chcemvediet/apps/inforequests/admin.py

```diff
@@ -20,7 +20,7 @@ class BranchInline(ReadOnlyAdminInlineMixin, admin.TabularInline):
     formset = BranchFormSet
     fields = [
             decorate(
-                lambda o: admin_obj_format(o),
+                lambda o: u'{} {}'.format(u'—' * (len(o.tree_order) - 1), admin_obj_format(o)),
```

Comment:
@martinmacko47 V pripade ak je branch hlavna medzdera medzi `{} {}` formatovacimi znakmi sa v prehliadaci nevyrederuje, takze som to neosetroval zvlast.

Reply:
@viliambalaz Namiesto `.format(...)` pouzivaj [`format_html(...)`](https://docs.djangoproject.com/en/3.0/ref/utils/), nech sa spravne escapuju stringy. Predsav si, ze namiesto `—` by si mal `<`, tak by vysledok bol neoescapovany.